README.rst

```diff
@@ -1,6 +1,9 @@
 Flake8 Coding plugin
 =====================
 
+.. image:: https://img.shields.io/pypi/v/flake8-coding.svg
```

Comment:
I think the package page on PyPI is not important for the repository visitors.
So the only meaning of the badge is showing release version.